test/model/content_test.dart

```diff
@@ -511,7 +510,7 @@ class ContentExample {
 
   static final mathInline = ContentExample.inline(
     'inline math',
-    r"$$ \lambda $$",
+    null, // r"$$ \lambda $$" (hypothetical server variation)
```

Comment:
I think you made this edit on a different example than intended :slightly_smiling_face:

Reply:
Ahh, should be fixed now.